run_action.py

```diff
@@ -20,6 +20,37 @@ def chunks(lst, n):
         yield lst[i : i + n]
 
 
+def markdown(s):
+    md_chars = "\\`*_{}[]<>()#+-.!|"
+
+
+    def escape_chars(s):
+        for ch in md_chars:
+            s = s.replace(ch, "\\" + ch)
+
+        return s
+
+
+    def unescape_chars(s):
+        for ch in md_chars:
+            s = s.replace("\\" + ch, ch)
+
+        return s
+
+
+    # Escape markdown characters
+    s = escape_chars(s)
+    # Decorate quoted symbols as code
+    s = re.sub(
+        "'([^']*)'",
+        lambda match:
+            "`` " + unescape_chars(match.group(1)) + " ``",
+        s
+    )
```

Comment:
Can't get my head around this. How does this work? Why are the characters unescaped?

Reply:
Also, why two `?

Reply:
Hi @platisd

> Can't get my head around this. How does this work? Why are the characters unescaped?

Because inside the code block all characters need to go as is. Consider:

`` `#include \<iostream\>` `` -> `#include \<iostream\>`

Backslashes are shown as is.

> Also, why two `?

To allow possible backticks in the code block, as above :)

Reply:
:bulb: makes sense! :}